### PR TITLE
Add Favicon and Title

### PR DESCRIPTION
**Pull request by amcguiga**
Made the favicon into the torch svg used for the header. Title is `Track 50a FOIL` to align with the eventual domain `track50a.astoria.digital`

https://github.com/astoria-tech/50-a-foil-status/issues/34 and https://github.com/astoria-tech/50-a-foil-status/issues/33

### CODE REVIEW

File: browser/public/index.html [hunk 1]
```diff
@@ -2,14 +2,14 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/torch.svg" />
```
Comment (bronzehedwick) [inline browser/public/index.html]: I think we should have a `favicon.ico` file as well for a fallback for non-supporting browsers, and/or adding the `rel="mask-icon"` for Safari. https://caniuse.com/?search=favicon%20svg

**Review by bronzehedwick — CHANGES_REQUESTED** [review]
This is great, thanks! Just comment for compatibility.

File: browser/public/index.html (hunk 1 above)
Comment (amcguiga) [reply browser/public/index.html]: Good call, I forgot how much of a headache favicons were. I'll be adding an svg and png backup and aligning it with different rels.

**Comment by amcguiga** [comment]
Built off of RealFaviconGenerator to scaffold out the different images for all the different browser stops. 
https://realfavicongenerator.net/favicon_result?file_id=p1f0a6frmq5c14u91b0u12cpuav6#.YEbCsJNKh2U

**Review by bronzehedwick — CHANGES_REQUESTED** [review]
Thanks so much @amcguiga, this is great! It does look like the `favicon.ico` file is empty...is that intentional, or a mistake in the auto favicon generator? Sorry to continue the favicon pain...

**Comment by amcguiga** [comment]
It's not empty, github is just considering it an `empty file` in diffs for some odd reason. If you look at it in the branch you'll see the raw image: https://github.com/astoria-tech/50-a-foil-status/blob/307c3f8ae4fa69e19206ab650eb410638eb79309/browser/public/favicon.ico

**Review by bronzehedwick — APPROVED** [review]

**Pull request merged**